test/parallel/test-https-autoselectfamily-slow-timeout.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const { request } = require('https');
+
+request(
+  'https://nodejs.org/en',
```

Comment:
Shouldn't this test go in test/internet/ if it's attempting to reach an internet host?

Reply:
Yes, I didn't know about it. Fixed now.